### ISSUE TITLE
Disable/Enable copilot and old context menu toggles

### ISSUE THREAD

**Issue by xe1st** [ChrisTitusTech/winutil#1725]
**Is your feature request related to a problem? Please describe.**
Not everyone wants ai stuff on their pc, especially I believe most of the powerusers

**Describe the solution you'd like**
Toggle for copilot

I guess these can be helpful for implementation
https://github.com/Atlas-OS/Atlas/tree/main/src/playbook/Executables/AtlasDesktop/4.%20Optional%20Tweaks/Windows%20Copilot

extra: old context menu (right click) toggle can also be implemented, because the modern one is just oversimplified and unintuitive imo: https://github.com/Atlas-OS/Atlas/tree/main/src/playbook/Executables/AtlasDesktop/4.%20Optional%20Tweaks/Windows%2011%20Context%20Menu

Credits to atlas os

**Comment by ourichermath** [comment]
I agree with this.  I just closed my browser and found 1/3 of my screen taken up with Copilot (my guess is Edge updated itself in the background and then ran Copilot).  I had to go to the Group Policy Editor to turn the thing off.

**Comment by ModernTTY** [comment]
> extra: old context menu (right click) toggle can also be implemented, because the modern one is just oversimplified and unintuitive imo: https://github.com/Atlas-OS/Atlas/tree/main/src/playbook/Executables/AtlasDesktop/4.%20Optional%20Tweaks/Windows%2011%20Context%20Menu

That already is in the toolbox in the tweaks section.

![IMG_2217](https://github.com/ChrisTitusTech/winutil/assets/165050080/4949cad8-5ad4-418c-a9f0-4303c6147050)

**Comment by SkylerWallace** [comment]
This can be added to disable Copilot from Taskbar:
```
Set-ItemProperty -Path "HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced" -Name "ShowCopilotButton" -Type "DWord" -Value "0"
```

Disable Copilot system-wide:
```
Set-ItemProperty -Path "HKLM:\SOFTWARE\Policies\Microsoft\Windows\WindowsCopilot" -Name "TurnOffWindowsCopilot" -Type "DWord" -Value "1"
```

Unfortunately I only found a group policy method for disabling Copilot system-wide, hopefully Microsoft adds a bonafide setting in the future.

**Comment by ModernTTY** [comment]
I am working on removing Copilot, PR coming soon it will completely uninstall Copilot not just unpin.

**Comment by Kyationa** [comment]
is it safe to do the classic right click menu?

**Comment by Real-MullaC** [comment]
> is it safe to do the classic right click menu?
> 
> 

Yes it is.

**Comment by cobaltjacket** [comment]
It would be useful to have options to disable in Windows, Office, and Edge separately or at once.

**Comment by daxliniere** [comment]
How's the disabling of Copilot coming along, @ChrisTitusTech ?

**Comment by isvein** [comment]
I installed W11 in an vm and tried everything to disable copilot, both the setting in winutil and the commands from this post, but copilot is still on the taskbar and I can start it manually.

Maybe just me who does not understand what this disable do?
Though it would also remove it from be manually started

**Comment by nonilevinson** [comment]
I had already disabled Copilot and I don't have any of its extensions, but unfortunately with the new version installed yesterday, Copilot has returned to the context menu when I'm in the Editor. From what I've read, MS is forcing us to use it and that's horrible. I would like to have the option to disable it again.

Version: 1.101.0 (system setup)
Commit: dfaf44141ea9deb3b4096f7cd6d24e00c147a4b1
Date: 2025-06-11T15:00:50.123Z
Electron: 35.5.1
ElectronBuildId: 11727614
Chromium: 134.0.6998.205
Node.js: 22.15.1
V8: 13.4.114.21-electron.0
OS: Windows_NT x64 10.0.19045

**Comment by ghost** [comment]
I dont know what this is a "Kepp Issue Open" both old context menu and disable copilot have been added /close
